src/server/google-cloud.md

```diff
@@ -42,14 +42,6 @@ To run Dart on clusters of Compute Engine instances,
 use Google Kubernetes Engine (GKE).
 
 For more information, see the [GKE overview][].
-
-## App Engine
-
-[App Engine][] support for Dart is incomplete,
-so we recommend that you **use Cloud Run** instead of App Engine for new
```

Comment:
I'd much rather explain what the recommendation is, rather than leave it undocumented.

It would also be good to synchronize terminology here and in https://pub.dev/packages/appengine's readme.